Rakefile

```diff
@@ -1,37 +1,3 @@
 require 'puppetlabs_spec_helper/rake_tasks'
-require 'puppet-lint/tasks/puppet-lint'
```

Comment:
This will need to be added to the .sync.yml to stop it from being removed, the following is a example:
```
Rakefile:
  requires:
    - puppet_blacksmith/rake_tasks
    - puppet-lint/tasks/puppet-lint
```

Reply:
Then you need to remember to run another `pdk convert` to distribute the changes.